msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AuthenticationResult.java

```diff
@@ -90,4 +90,6 @@ private ITenantProfile getTenantProfile() {
     private final Date expiresOnDate = new Date(expiresOn * 1000);
 
     private final String scopes;
+
+    private final Boolean isPopAuthorization;
```

Comment:
Is this used?

Reply:
Yes, it can be set when creating the AuthenticationResult in the IBroker file: https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/8363f81cfd942e269758e9af1e2c2c1c53fca275/msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/IBroker.java#LL83C20-L83C20

(the field here is 'private', but we use a dependency called Lombok to generate the getters/setters and other boilerplate code for these fields)